infrastructure/stacks/development-pipeline/unit-tests-buildspec.yml

```diff
@@ -1,75 +1,15 @@
+# For documentation see here - https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html
 version: 0.2
 
-#env:
-  #variables:
-    # key: "value"
-    # key: "value"
-  #parameter-store:
-    # key: "value"
-    # key: "value"
-  #secrets-manager:
-    # key: secret-id:json-key:version-stage:version-id
-    # key: secret-id:json-key:version-stage:version-id
-  #exported-variables:
-    # - variable
-    # - variable
-  #git-credential-helper: yes
-#batch:
-  #fast-fail: true
-  #build-list:
-  #build-matrix:
-  #build-graph:
 phases:
-  install:
-    runtime-versions:
-      docker: 19
-  #install:
-    #If you use the Ubuntu standard image 2.0 or later, you must specify runtime-versions.
-    #If you specify runtime-versions and use an image other than Ubuntu standard image 2.0, the build fails.
-    #runtime-versions:
-      # name: version
-      # name: version
-    #commands:
-      # - command
-      # - command
-  pre_build:
-    commands:
-      - echo $CODEBUILD_RESOLVED_SOURCE_VERSION
-      - make wait-for-codebuild-to-finish PROJECT_NAME=$TESTER_BUILD_PROJECT_NAME
-      - aws sts get-caller-identity
-      - temp_role=$(aws sts assume-role --role-arn "arn:aws:iam::$AWS_ACCOUNT_ID_NONPROD:role/UECPUPipelineBuildRole" --role-session-name "CodeBuildSession")
-      - export AWS_ACCESS_KEY_ID=$(echo $temp_role | jq -r .Credentials.AccessKeyId)
-      - export AWS_SECRET_ACCESS_KEY=$(echo $temp_role | jq -r .Credentials.SecretAccessKey)
-      - export AWS_SESSION_TOKEN=$(echo $temp_role | jq -r .Credentials.SessionToken)
-      - aws sts get-caller-identity
-      - chown -R 999:999 $CODEBUILD_SRC_DIR
-      - make docker-hub-signin
-
   build:
     commands:
-      - aws sts get-caller-identity
-      - echo "[Running Unit Tests]"
-      - make docker-pull NAME=tester VERSION=latest
-      - make unit-test
-      # - command
-  #post_build:
-    #commands:
-      # - command
-      # - command
+      - cd application
+      - python -m pytest --junitxml=./testresults.xml --cov-report term-missing  --cov-report xml:coverage.xml --cov=. -vv
```

Comment:
could use `make unit-test-local`

Reply:
The make target builds a venv environment which isn't required for this codebuild stage

Reply:
The make target builds a venv environment which isn't required for this codebuild stage